addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21275,3 +21275,10 @@ msgstr ""
 msgctxt "#39110"
 msgid "Select Resolution"
 msgstr ""
+
+#. audio/video/subtilte flag - FLAG_ORIGINAL
+#: xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp
+#: xbmc/video/dialogs/GUIDialogVideoSettings.cpp
+msgctxt "#39111"
+msgid "OV"
```

Comment:
Is "OV" intended as a placeholder some other string?

Reply:
I guess many users will not know what OV is. See comment from @scott967. Would make sense not to use an abbreviation here, imo.

Reply:
Its displayed in audio stream dialog and should be short / IMO its a well known shortcut for Original version, at least in germany we use this denglish thing too.